Simplify notification wording

danxuliu opened the pull request:
[As suggested by @jancborchardt](https://github.com/nextcloud/spreed/pull/500#issuecomment-356440981); although the meaning is not exactly the same (personally I would add back the _now_) the new wording is simpler and thus easier to understand.

jancborchardt commented:
Let’s go even simpler?

> You were mentioned by **username** (now deleted) in **Company call**

(Of course the (now deleted) addition will only come if  that user has been deleted.)

danxuliu commented:
>  **username** (now deleted) mentioned you in **Company call**

Funny thing is that [we are almost going back to the original wording](https://github.com/nextcloud/spreed/pull/500/files#diff-52c5c003756b8dcc9c5318257107346dL198) (which I think makes sense, as it is more consistent with the rest of messages).

Note that when the user is deleted her name is no longer known, so in that case the message would be
> A user (now deleted) mentioned you in **conversation name**

or something like that.

In the same way the room may have no name, so that is why current messages are
>  **username** mentioned you in a group conversation
>  **username** mentioned you in a group conversation: **conversation name**

If the message for deleted users is changed to 
>  A user (now deleted) mentioned you in **conversation name**
>  A user (now deleted) mentioned you in a group conversation

it would make sense to change that too for mentions by guests and users not deleted.

Personally I would keep the current _in a group conversation_ / _in a group conversation: **conversation name**_ also for deleted users (I somehow feel (so do not ask me for examples, it is just a feeling :-P ) that, depending on the conversation name, _embedding_ it could lead to unclear notification messages), but your call ;-)

mario commented:
I would keep current messages too atm.

On Thu, 31 May 2018 at 12:32, Daniel Calviño Sánchez <
notifications@github.com> wrote:

> *username* (now deleted) mentioned you in *Company call*
>
> Funny thing is that we are almost going back to the original wording
> <https://github.com/nextcloud/spreed/pull/500/files#diff-52c5c003756b8dcc9c5318257107346dL198>
> (which I think makes sense, as it is more consistent with the rest of
> messages).
>
> Note that when the user is deleted her name is no longer known, so in that
> case the message would be
>
> A user (now deleted) mentioned you in *conversation name*
>
> or something like that.
>
> In the same way the room may have no name, so that is why current messages
> are
>
> *username* mentioned you in a group conversation
> *username* mentioned you in a group conversation: *conversation name*
>
> If the message for deleted users is changed to
>
> A user (now deleted) mentioned you in *conversation name*
> A user (now deleted) mentioned you in a group conversation
>
> it would make sense to change that too for mentions by guests and users
> not deleted.
>
> Personally I would keep the current *in a group conversation* / *in a
> group conversation: conversation name* also for deleted users (I somehow
> feel (so do not ask me for examples, it is just a feeling :-P ) that,
> depending on the conversation name, *embedding* it could lead to unclear
> notification messages), but your call ;-)
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nextcloud/spreed/pull/928#issuecomment-393488498>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAAWsuRfbi1eiZGjWIGcGFvQSWWEbEUsks5t38bAgaJpZM4UToOY>
> .
>

jancborchardt commented:
Folks, we need to keep it short for mobile notifications. This stuff is not visible.
